src/webgl/material.js

```diff
@@ -572,8 +604,9 @@ p5.prototype.textureWrap = function(wrapX, wrapY = wrapX) {
 };
 
 /**
- * Ambient material for geometry with a given color. You can view all
- * possible materials in this
+ * Ambient material for geometry with a given color. Ambient material defines the color the object reflects under any lighting.
+ * For example, if the ambient material of an object is pure red, but the ambient lighting only contains green, the object will not reflect any light. 
+ * You can view all possible materials in this
```

Comment:
Can you update this to read:
`Here's an <a href="https://p5js.org/examples/3d-materials.html">example containing all possible materials</a>.`
This will make it easier to read via screen reader device.